Remotion/Core/Reflection.CodeGeneration/DPExtensions/ExpressionReference.cs

```diff
@@ -65,7 +65,7 @@ private LocalReference CreateLocal (ILGenerator gen)
       return local;
     }
 
-    public override void StoreReference (ILGenerator gen)
+    public override void StoreReference (ILGenerator? gen)

```

Comment:
so, notnull and a nfo in tests?

Reply:
yes. impl is not supported, value in test is probably a stub (I'm assuming) with ignored argument values. in generel: wenn the test passes dmmy values because IgnoreArguments, those are not indicative of the signatures nullablity.